### PR TITLE
Todo app

### PR DESCRIPTION
**Pull request by edemagbenyo**
Changes implemented:

1. Create, Update and Delete a todo
2. Create a Project
3. List each todo under a project

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <div class="m-5">
+      <h1 class="text-danger welcome text-center ">Organize your thoughts in One Place</h1>
+    </div>
+  </header>
+
+  <div class="container-fluid">
+    <div class="row">
+
+      <div class="col-md-3 jumbotron">
+        <button class="btn btn-danger mt-3" id="btnProject">Add Project</button>
```
Comment (Torres-ssf) [inline dist/index.html:29]: Please let me ask, why do you have two `Add Projects` buttons and also 2 `Add tasks` buttons in your HTML file? :thinking:

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "todoapp",
+  "version": "1.0.0",
+  "description": "A simple TODO app",
+  "main": "index.js",
+  "scripts": {
+    "watch": "webpack --watch --mode development",
```
Comment (Torres-ssf) [inline package.json]: Maybe it makes more sense to change this script name to dev since its purpose it's for development.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "build": "webpack --mode production",
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (Torres-ssf) [inline package.json:9]: Since no tests were implemented we can safely delete the test script :smiley:

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :warning: 
Hi there team :hand: 
I must say, you did an awesome work in this project. It sure looks fine :clap: :+1: 

However, we have some changes to make before moving on.

- [x] I notice you forgot to summarize the changes made in the PR description. This is a good practice that will make your PR look more professional. Please take a look [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) to see how you can make your repo looks more professional.
Screenshot from 2020-03-27 22-07-39

- [ ] Good job on your README file, it's looking really nice. But I notice you forgot to add instructions on how to use your app and the scripts it uses. You can benefit from that since it's a good practice and will make your repo look more professional.

- [ ] The main.js file inside the dist directory works fine, all functionalities are working nice and the save to local storage as well. But when I run `npm run build` or `npm run watch`, the new generated main.js seems to stop working properly. Maybe this is happening only on my end. Can you confirm that?
problem

Please kindly submit for code review again after making these changes.
Feel free to message me on slack if you have any question

Happy coding :computer: 
**Sergio Torres**

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle:

Hello,
Thank you for the improvements you've done !
However, there are some bugs that should be fixed.

1. The TODO creation form doesn't have any field for the Project is assigned to

2. There is an error when trying to **edit** a created TODO
Screenshot from 2020-03-30 18-06-12

3. There is also that error met when using the checkbox above the TODO
Screenshot from 2020-03-30 18-08-10

4. After submiting the Project creation form, the project's name is still into the field

5. You can't handle TODO and Project into the same file. Please, re-read this [article](http://aspiringcraftsman.com/2011/12/08/solid-javascript-single-responsibility-principle/) about SOLID

6. Also, The user should be able to filter the TODO by project.

Kindly make the suggested changes and submit a new code review request,
Thanks

Mael FOSSO

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap:

You can merge it :+1:
Good luck with the next step :handshake:

Mael FOSSO

**Pull request merged**